docs/source/API/core/MultiGPUSupport.rst

```diff
@@ -0,0 +1,153 @@
+.. role:: cppkokkos(code)
+    :language: cppkokkos
+
+Multi-GPU Support
+=================
+
+Kokkos has support for launching kernels on multiple GPUs from a single host process, e.g., a single MPI rank. This feature currently
+exists for the CUDA, HIP, and SYCL backends.
+
+Using this feature requires knowledge of backend specific API calls for creating non-default execution space instances. Once the execution space has been
+created, it can be used to create execution policies, allocate views, etc. all on the device chosen by the user.
+
+Constructing Execution Spaces
+-----------------------------
+
+CUDA
+~~~~
+
+For CUDA backend, the user creates a ``cudaStream_t`` object and passes it to the ``Kokkos::Cuda`` constructor.
+
+.. note:: The lifetime of all Kokkos objects related to the stream (e.g., the execution space) needs to end before the stream itself is invalidated
+          via ``cudaStreamDestroy``. This can be done by adding a scope around such objects, as is done below.
+
+
+.. code-block:: cpp
+
+    // Query number of devices available
+    int n_devices;
+    cudaGetDeviceCount(&n_devices);
+
+    // Choose some 0 <= N < n_devices
+    int N = ...;
+
+    // Create stream on device N
+    cudaStream_t stream;
+    cudaSetDevice(N);
+    cudaStreamCreate(&stream);
+
+    // Scope execution space to ensure stream
+    // is destroyed *after* execution space
+    {
+      // Create execution space
+      Kokkos::Cuda exec_space(stream);
+
+      // Use execution space
+      /* ... */
+    }
+
+    // Destroying stream after use
+    cudaSetDevice(N);
+    cudaStreamDestroy(stream);
+
+HIP
+~~~
+
+For the HIP backend, like with CUDA, the user creates a ``hipStream_t`` object and passes it to the ``Kokkos::HIP`` constructor.
+
+.. note:: The lifetime of all Kokkos objects related to the stream (e.g., the execution space) needs to end before the stream itself is invalidated
+          via ``cudaStreamDestroy``. This can be done by adding a scope around such objects, as is done below.
```

Comment:
```suggestion
.. note:: The lifetime of all Kokkos objects related to the stream (e.g., the execution space) needs to end before the stream itself is invalidated
          via ``hipStreamDestroy``. This can be done by adding a scope around such objects, as is done below.
```
I guess a copy-pasta error